### PR TITLE
Feature/consensus

### PR DESCRIPTION
**Pull request by lllwvlvwlll**

### CODE REVIEW

File: en-us/sc/consensus.md [hunk 1]
```diff
@@ -5,12 +5,15 @@ name: Consensus
...
-* Proof of Stake (PoS) - A schema which uses network consensus to handle fault tolerance.
-* Proof of Work (PoW) - A schema which uses compute utility to handle fault tolerance.
-* Byzantine Fault (BF) - A failure in which a node continues to operate in a non-honest manner.
-* Practical Byzantine Fault Tolerance (PBFT) - A cosensus algorithm implemented within the AntShares block chain to garantee fault tolerance.
-* View - The data set used during consensus in AntShares PBFT
+* **Proof of Work** `PoW` - A schema which uses compute utility to handle fault tolerance.
+
+* **Byzantine Fault** `BF` - A failure in which a node continues to operate in a non-honest manner.
+
+* **Practical Byzantine Fault Tolerance** `PBFT` - A cosensus algorithm implemented within the AntShares block chain to garantee fault tolerance.
```
Comment (fabwa) [inline en-us/sc/consensus.md]: They actually call it dBFT which is a version of pBFT

**Comment by erikzhang** [comment]
so many commits

**Comment by canesin** [comment]
nice, on related @lllwvlvwlll I added readthedocs at your request... I believe this can go on forever.. so in my view it is already enough for the interested one to dive in. Let's put out there and see it improved... @erikzhang he will squash merge when accepting this PR before, did you review ?

**Comment by lllwvlvwlll** [comment]
centered it for you ;)

I will wait for erik's approval on this one since I want to make sure it doesn't misrepresent.

**Comment by fabwa** [comment]
http://pmg.csail.mit.edu/papers/osdi99.pdf for 2nd ref - also use [1][2][3] in running text for reference

**Comment by Ejhfast** [comment]
Awesome job on this! Only super minor quibble: if it was academic work, you would have references appearing in text

**Pull request closed without merging**